File: languages/javascript/examples/basic.js
"use strict";

const { Generate, GenerateMarkerComplete, Validate } = require("../src/position-tape");

const exact = Generate(100);
const markerComplete = GenerateMarkerComplete(1000);
const validation = Validate(exact, 100);

console.log(exact);
console.log(markerComplete.length);
console.log(validation.isValid);

File: languages/lua/examples/basic.lua
package.path = "languages/lua/src/?.lua;" .. package.path

local pt = require("position_tape")

local exact = pt.Generate(100)
local marker_complete = pt.GenerateMarkerComplete(1000)
local validation = pt.Validate(exact, 100)

print(exact)
print(#marker_complete)
print(validation.is_valid)
